testsuite/tests/kuadrant/authorino/identity/rhsso/conftest.py

```diff
@@ -10,13 +12,34 @@ def oidc_provider(rhsso):
 
 
 @pytest.fixture(scope="module")
-def authorization(authorization, rhsso):
+def authorization(authorization, rhsso, jwt_ttl):
     """Add RHSSO identity to AuthConfig"""
-    authorization.identity.add_oidc("rhsso", rhsso.well_known["issuer"])
+    authorization.identity.add_oidc(
+        "rhsso",
+        rhsso.well_known["issuer"],
+        ttl=jwt_ttl,
+    )
     return authorization
 
 
 @pytest.fixture(scope="module")
 def realm_role(rhsso, blame):
     """Creates new realm role"""
     return rhsso.realm.create_realm_role(blame("role"))
+
+
+@pytest.fixture(scope="module")
+def jwt_ttl():
+    """Returns TTL in seconds for Authorino to trigger OIDC discovery and update its cache"""
+    return 30
+
+
+@pytest.fixture(scope="module")
+def create_jwt(rhsso, auth):
```

Comment:
`create_jwt_auth`

Reply:
Make sense, will rename as you suggested